tools/crm_mon.c

```diff
@@ -445,14 +452,12 @@ show_attributes_cb(const gchar *option_name, const gchar *optarg, gpointer data,
 static gboolean
 show_bans_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **error) {
     if (optarg != NULL) {
-        if (options.neg_location_prefix != NULL) {
-            free(options.neg_location_prefix);
-        }
-
-        options.neg_location_prefix = strdup(optarg);
+        char *s = crm_strdup_printf("bans:%s", optarg);
+        return include_exclude_cb("--include", s, data, error);
+        free(s);
```

Comment:
Not much point in freeing after return :)

Reply:
Hah, no.  Something tells me this is not the last time I'll do this.  Incidentally, -Wunreachable-code no longer does anything.